i18n/dnd5e/gl/dnd5e.json

```diff
@@ -0,0 +1,758 @@
+  "DND5E.WeaponPropertiesRet": "Recarga",
+  "DND5E.WeaponPropertiesSil": "Prateada",
+  "DND5E.WeaponPropertiesSpc": "Especial",
+  "DND5E.WeaponPropertiesThr": "Proxectil",
+  "DND5E.WeaponPropertiesTwo": "Dúas Mans",
+  "DND5E.WeaponPropertiesVer": "Versátil",
+  "DND5E.WeaponSiege": "Arma de Cerco",
+  "DND5E.WeaponSimpleM": "Simple Corpo a Corpo",
+  "DND5E.WeaponSimpleProficiency": "Armas Simple",
+  "DND5E.WeaponSimpleR": "Simple a Distancia",
+  "DND5E.Weight": "Peso",
+  "DND5E.available": "disponíbel",
+  "DND5E.description": "Un sistema para xogar á quinta edición do RPG de Mesa mais popular do mundo no ambiente Foundry Virtual Tabletop.",
+  "DND5E.of": "de",
+  "DND5E.per": "por",
+  "DND5E.spell": "será conxurada",
+  "SETTINGS.5eAllowPolymorphingL": "Permitir a xogadores polimorfaren os seus propios actores.",
+  "SETTINGS.5eAllowPolymorphingN": "Permitir Polimorfia",
+  "SETTINGS.5eAutoCollapseCardL": "Fechar automaticamente descricións de Cartóns de Item no rexistro de chat",
+  "SETTINGS.5eAutoCollapseCardN": "Fechar Cartóns de Item no chat",
+  "SETTINGS.5eAutoSpellTemplateL": "Cando unha maxia fose conxurada, por padrón inicia o proceso de crear un Modelo de Medición correspondente se o hai (require cargo de xogador DE FIAR ou superior)",
+  "SETTINGS.5eAutoSpellTemplateN": "Sempre empregar Modelos de Maxia",
+  "SETTINGS.5eCurWtL": "Moedas cargadas afectan a sobrecarga do personaxe seguindo as regras no LDX px. 143 (EN).",
+  "SETTINGS.5eCurWtN": "Aplicar peso de moedas",
+  "SETTINGS.5eDiagEuclidean": "Euclidiano (Diagonal 7.07 ft.)",
+  "SETTINGS.5eDiagL": "Configura que regra de movemento diagonal debe ser usada para xogos neste sistema.",
+  "SETTINGS.5eDiagN": "Regra de Movimento Diagonal",
+  "SETTINGS.5eDiagPHB": "LDX: Equidistante (5/5/5)",
+  "SETTINGS.5eDiagDMG": "GDM: Alternando (5/10/5)",
+  "SETTINGS.5eInitTBL": "Adiciona o valor bruto do atributo Destreza para desempatar roladas de Iniciativa.",
+  "SETTINGS.5eInitTBN": "Desempate de Iniciativa usando Destreza",
+  "SETTINGS.5eNoExpL": "Elimina a barra de experiencia das fichas de personaxe.",
+  "SETTINGS.5eNoExpN": "Desactivar rexistro de experiencia",
+  "SETTINGS.5eRestL": "Configura que regra variante de descanso debe ser usada para xogos neste sistema.",
+  "SETTINGS.5eRestN": "Variante de Descanso",
+  "SETTINGS.5eRestPHB": "Libro do Xogador (DL: 8 horas, DC: 1 hora)",
+  "SETTINGS.5eRestGritty": "Realismo Robusto (DL: 7 dias, DC: 8 horas)",
+  "SETTINGS.5eRestEpic": "Heroísmo Épico (DL: 1 hora, DC: 5 min)",
+  "DND5E.ActorWarningInvalidItem": "Items do tipo {itemType} non poden ser engadidos a un(ha) {actorType}.",
+  "DND5E.AddEmbeddedItemPromptHint": "Queres engadir estes items á túa folla de personaxe?",
+  "DND5E.Apply": "Aplicar",
+  "DND5E.ClassMakeOriginal": "Clase Orixinal",
+  "DND5E.ClassMakeOriginalHint": "Primeira clase escollida polo personaxe usada para determinar certos trazos de clase cando se fai multiclase.",
```

Comment:
Penso que queda máis clariño con verbo: "A primeira clase escollida pola personaxe é usada..."